File: Atividade_js/JoãoFulano.js
function ValorPresente(fv, taxa, anos) {
    return fv / Math.pow(1 + taxa, anos);
}

const taxajuros = 0.08;

const valor_futuro = 12000;
const anosfulano = 10;
const valor_depositar = ValorPresente(valor_futuro, taxajuros, anosfulano);

console.log(`joão fulano precisa depositar hoje: $ ${valor_depositar.toFixed(2)}`);

File: Atividade_py/JoãoFulano.py
taxa_juros = float(input("digite a taxa de juros anual: ")) / 100

valor_futuro = 12000  
anos = 10
valor_presente = valor_futuro / (1 + taxa_juros) ** anos

print(f"joão fulano precisa depositar hoje R$ {valor_presente:.2f} para ter R$ 12000 em 10 anos.")

